py/selenium/webdriver/firefox/extension_connection.py

```diff
@@ -65,7 +65,7 @@ def connect(self):
 
     @classmethod
     def connect_and_quit(cls):
-        """Connects to an running browser and quit immediately."""
+        """Connects to a running browser and quit immediately."""
```

Comment:
I would suggest "Quits"

Connects to a running browser and quits immediately

Reply:
@diemol we should be able to just delete this class. Pretty sure this is referring to the Selenium 2 Firefox extension :joy: